tests/plugin/text/intent_entity_mutator/mutation_rules.yaml

```diff
@@ -0,0 +1,556 @@
+swap_rules:
+  - mutate: intent # Can only mutate intent or entity
+    mutate_to: "bill_paid"
+    conditions:
+      state:
+        in: ["ASK_BILL_PAID_DATE", "CONFIRM_BILL_PAID_DATE"]
+        not_in: []
```

Comment:
Can't we make keys with empty values as optional? This includes keys like `in` and `not_in` but also primitives like `previous_intent`, `transcript`, etc.

Reply:
We can. I have just kept to maintain a structure of the `rules` file.